blog/2023-02-04-mysql-slack-blog/index.md

```diff
@@ -0,0 +1,246 @@
+target: http://192.168.49.2:30001/gateway/mysql-to-slack # Vanus in Playground
+name: "quick_start"
+db:
+  host: "MYSQL_HOST" # IP address of MySQL server
+  port: MYSQL PORT # port address of MySQL server
+  username: "vanus" # Username 
+  password: "PASSWORD" # Password previously set
+database_include: [ "<DATABASE_NAME>" ] # The name of your database
+# format is vanus_test.tableName
+table_include: [ "TABLE_NAME" ] # The name of your Table
+
+store:
+  type: FILE
+  pathname: "/vanus-connect/data/offset.dat"
+
+db_history_file: "/vanus-connect/data/history.dat"
+EOF
+```
+11. Run MySQL Source in the background
+
+```shell
+docker run -it --rm --network=host \
+  -v ${PWD}/mysql:/vanus-connect/config \
+  -v ${PWD}:/vanus-connect/data \
+  --name source-mysql public.ecr.aws/vanus/connector/source-mysql &
+```
+
+12. Create an app on Slack. Visit Slack Website
+
+![slack ](img/clickcreate.gif)
+
+13. Click 'create new app' and from scratch
+
+![slack ](img/scratch.gif)
+
+14. Set the bot name as "Vanus-MySQL"
+15. Chose your workspace
+
+![slack ](img/entername.gif)
+
+16. Click on permissions in the central menu.
+
+![slack ](img/permission.gif)
+
+17. Set bot permissions  'Add OAuth Scope' chat:write and chat:write.public.
+
+![slack ](img/scopes.gif)
+
+18. Install the bot to your workspace.
+
+![slack ](img/installchannel.gif)
+
+19. Keep your Bot User OAuth Token in OAuth & Permissions.
+
+![slack ](img/token.gif)
+
+20. Create a yml file named slack-sink.yml in the playground with the following command:
+
+```shell
+cat << EOF > slack-sink.yml
+apiVersion: v1
+kind: Service
+metadata:
+  name: sink-slack
+  namespace: vanus
+spec:
+  selector:
+    app: sink-slack
+  type: ClusterIP
+  ports:
+    - port: 8080
+      name: sink-slack
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: sink-slack
+  namespace: vanus
+data:
+  config.yml: |-
+    default: "test-app1"
+    slack:
+      - app_name: "test-app1"
+        token: "xoxb-429"
+        default_channel: "#general"
+
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: sink-slack
+  namespace: vanus
+  labels:
+    app: sink-slack
+spec:
+  selector:
+    matchLabels:
+      app: sink-slack
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: sink-slack
+    spec:
+      containers:
+        - name: sink-slack
+          image: public.ecr.aws/vanus/connector/sink-slack
+          resources:
+            requests:
+              memory: "128Mi"
+              cpu: "100m"
+            limits:
+              memory: "512Mi"
+              cpu: "500m"
+          imagePullPolicy: Always
+          ports:
+            - name: http
+              containerPort: 8080
+          volumeMounts:
+            - name: config
+              mountPath: /vanus-connect/config
+      volumes:
+        - name: config
+          configMap:
+            name: sink-slack
+EOF
+```
+
+21. Update the settings: `Channel` and `token`
+
+```shell
+vi slack-sink.yml
+```
```

Comment:
Need to add the command to run slack-sink.yml
kubectl apply -f slack-sink.yml

Reply:
thanks